src/restconfig/src/main/java/org/geoserver/rest/resources/resourceMetadata.ftl

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
```

Comment:
I am assuming that the infrastructure will set the charset when the response is returned; does it add a meta header?

Is correct to not add a charset here for the freemarker template itself? Or do we insist that resources are UTF-8? I think Maven is configured to do so. We might need <#ftl encoding="utf-8"> in this file.
http://freemarker.incubator.apache.org/docs/pgui_misc_charset.html

I am unsure whether this file should contain <meta charset="utf-8"> after ```<head>```.